src/components/App/App.js

```diff
@@ -5,36 +5,60 @@ class App extends Component {
     super()
     this.state = {
       colors: []
+    };
+  };
+
+  componentDidMount(){
+    this.makeFiveColors()
+  };
+
+  makeFiveColors = () => {
+    let colors = []
+    for (let i = 0; i < 5; i++) {
+      let color = this.genHex()
+      colors.push(color)
     }
-  }
+    this.setState({colors})
+  };
+
+  genHex = () => {
+    let hexables = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'A', 'B', 'C', 'D', 'E', 'F' ];
+    let shuffledHexables = hexables.sort(() => Math.random() - 0.5)
+    let newColor = shuffledHexables.reduce((finalColor, hexable) => {
+        finalColor += hexable
+        return finalColor.slice(0, 6);
```

Comment:
So hexables array are being sorted randomly and once is sorted we are making a new array with only 6 indices.